.woodpecker.yml

```diff
@@ -53,7 +54,10 @@ steps:
     group: format
     image: backplane/pgformatter:latest
     commands:
-      - ./scripts/sql_format_check.sh
+      - apt install git
+      - find migrations -type f -name '*.sql' -exec pg_format -i {} +
+      # ensure that formatting doesn't result in any git changes
+      - git diff --exit-code
```

Comment:
git isn't the right way to handle this, because it would fail on a new migration. I'd recommend just editing the format_check.sh function to use find exec.

Reply:
No thats fine, git diff will only give exit code 1 if there are uncommitted changes. So if you add migrations in your PR they are part of a commit, and exit code is 0.

Find exec is complicated because then you need a different temp file for each input. Its annoying that pg_format doesnt have a --check option.

Reply:
But how is it going to make sure the new migration that got added is correct sql?

Reply:
Another easy way I can think of to speed it up, is in the sql_format_check script, use gnu parallel and make sure the tmp file is a new random one each time.